doc/source/tracking/paper.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'icepyx'
+tags:
+  - Python
+  - ICESat-2
+  - LiDAR
+  - elevation
+  - community
+  - cloud
+authors:
+  - name: Jessica Scheick^[Corresponding author] # note this makes a footnote
+    orcid: 0000-0002-3421-4459
+    affiliation: 1
+  - name: Wei Ji Leong
+    orcid: 0000-0003-2354-1988
+    affiliation: 2
+  - name: Kelsey Bisson
+    affiliation: 3
+  - name: Anthony Arendt
+    affiliation: 4
+  - name: Shashank Bhushan
+    affiliation: 4
+  - name: Zach Fair
+    affiliation: 5
+  - name: Raphael Hagen
+    affiliation: 4
+  - name: Sarah Hall
+    affiliation: 1  
+  - name: Scott Henderson
+    affiliation: 4
+  - name: Tom Johnson
+    affiliation: 6
+  - name: Friedrich Knuth
+    affiliation: 7
+  - name: Tian Li
+    affiliation: 8
+  - name: Zheng Liu
+    affiliation: 4
+  - name: Luis Lopez
+    affiliation: 9
+  - name: Romina Piunno
+    affiliation: 10
+  - name: Nitin Ravinder
+    affiliation: 11
+  - name: Landung "Don" Setiawan
+    affiliation: 4
+  - name: Trey Stafford
+    affiliation: 9
+  - name: Amy Steiker
+    affiliation: 9
+  - name: Tyler Sutterley
+    orcid: 0000-0002-6964-1194
+    affiliation: 4
+  - name: JP Swinski
+    affiliation: 12
+  - name: Bruce Wallin
+    affiliation: 9
+  - name: Learn2Pheonix
+    affiliation: 13
+
+# Note: first three authors are driving the publication. Additional contributors/authors are listed in alphabetical order by last name. Anyone who also contributes substantially to preparing the JOSS submission will be moved into ABC order after the first three and before the non-publication contributors to icepyx. Non-responsive coauthors will be removed from the list since their permission to be included was not granted.
+  
```

Comment:
@weiji14 Does this answer your question? Happy to have a more in depth conversation about it (that goes for anyone!).

Reply:
Ah yes, that's a good answer. There's also some notes on https://joss.readthedocs.io/en/latest/submitting.html#authorship which is worth a read.